.travis/brew_install_gcc6.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
The file name for this file can probably be renamed to "install_osx_dependencies.sh"